Description: pybuild-based install process
 This patch deploy a small setup.py file in the root
 directory of the sources to support pybuild-based installation
 process.
Bug: https://github.com/kamorin/DHCPig/issues/13
Author: Philippe Thierry
Last-Update: 2017-06-16
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+
+# Authors:
+#   Philippe Thierry <phil@reseau-libre.net>
+
+import os
+import os.path
+
+from distutils.core import setup
+
+setup(
+    name="dhcpig",
+    author="Kevin Amorin",
+    description="DHCP exhaustion script using scapy network library",
+    license="GPL2+",
+    url="https://github.com/kamorin/DHCPig",
+    scripts=[
+        ("pig.py")
+    ],
+)
